frontend/components/auction/collateral/MarketPriceSelection.vue

```diff
@@ -82,6 +82,25 @@ export default Vue.extend({
             }
             return undefined;
         },
+        callees(): Object {
+            const unknown = [];
+            const sorted = [];
+            for (const callee in this.auctionTransaction.marketData) {
+                if (
+                    this.auctionTransaction.marketData[callee].unitPrice &&
+                    !this.auctionTransaction.marketData[callee].unitPrice.isNaN()
+                ) {
+                    sorted.push([callee, this.auctionTransaction.marketData[callee].unitPrice]);
+                } else {
+                    unknown.push([callee, this.auctionTransaction.marketData[callee].unitPrice]);
+                }
+            }
+            sorted.sort((a, b) => a[1].minus(b[1]).toNumber());
+            const calleesArray = [...sorted, ...unknown];
+            const calleesObject = {};
+            calleesArray.forEach(item => (calleesObject[item[0]] = this.auctionTransaction.marketData[item[0]]));
+            return calleesObject;
```

Comment:
There is no reason to convert this back to the object. The problem is the controversy that JS does not guarantee object key order. Suggestion: please return `[...sorted, ...unknown]` from this function and update your template to match